app/proguard-benchmark-rules.pro

```diff
@@ -0,0 +1 @@
+-keep class androidx.datastore.*.** {*;}
```

Comment:
I'm not seeing this file be used in the `benchmark` build type in `build.gradle.kts`. Is it actually necessary?